advanced/matrix/test_matrix.py

```diff
@@ -0,0 +1,120 @@
+"""Unit testing for the Matrix class."""
```

Comment:
**Change all methods so that every test case would check only one case. (one assert). If you want to check a few test cases of add_matrix for example, then write few different unit test methods checking every case separately.**